src/mapcraftercore/mc/chunk.cpp

```diff
@@ -178,15 +178,18 @@ bool Chunk::readNBT(mc::BlockStateRegistry& block_registry, const char* data, si
 		}
 	}
 
-	if (level.hasArray<nbt::TagByteArray>("Biomes", 256)) {
+	if (level.hasArray<nbt::TagByteArray>("Biomes", BIOMES_ARRAY_SIZE)) {
 		const nbt::TagByteArray& biomes_tag = level.findTag<nbt::TagByteArray>("Biomes");
 		std::copy(biomes_tag.payload.begin(), biomes_tag.payload.end(), biomes);
-	} else if (level.hasArray<nbt::TagIntArray>("Biomes", 256)) {
+	} else if (level.hasArray<nbt::TagIntArray>("Biomes", BIOMES_ARRAY_SIZE)) {
 		const nbt::TagIntArray& biomes_tag = level.findTag<nbt::TagIntArray>("Biomes");
 		std::copy(biomes_tag.payload.begin(), biomes_tag.payload.end(), biomes);
 	} else if (level.hasArray<nbt::TagByteArray>("Biomes", 0)
 			|| level.hasArray<nbt::TagLongArray>("Biomes", 0)) {
-		std::fill(biomes, biomes + 256, 0);
+		std::fill(biomes, biomes + BIOMES_ARRAY_SIZE, 0);
+	} else if (level.hasArray<nbt::TagByteArray>("Biomes", 256) || level.hasArray<nbt::TagIntArray>("Biomes", 256)) {
+		LOG(WARNING << "Out dated chunk " << chunkpos << ": Old biome data found!";
+	}
```

Comment:
Missing ")" after "LOG(WARNING"
And the "}" is one too much

Reply:
Dang, I thought I'd fixed that one before sending it out